lib/serverHandler.dart

```diff
@@ -256,6 +257,10 @@ class ServerHandler {
       Map<String, dynamic> info = data["info"];
 
       updateLocalPlanet(player, info);
+    } else if (action == "respawnTimerUpdate") {
```

Comment:
Let's keep the format 'objectChanged', so 'respawnTimerUpdated', just for consistency.